orangecontrib/network/widgets/OWNxExplorer.py

```diff
@@ -375,6 +375,7 @@ def set_node_data(self, data):
     @Inputs.node_subset
     def set_node_subset(self, data):
         super().set_subset_data(data)
+        super()._handle_subset_data()
```

Comment:
This must be called from `handleNewSignals`. It may happen that the widget receives the subset first, and then the data. Method `handleNewSignals` is called when the widget receives all expected signals.

`_handle_subset_data` is actually called in the inherited `handleNewSignals`, but OWNxExplorer's overrides it without calling the inherited method. We are going to have the same problem whenever the base class adds something to `handleNewSignals`.

Please either move this call to `handleNewSignals` (as a temporary solution) or, if you are willing to dig deeper, see if OWNxExplorer could call the inherited method.  The OWMDS widget is similar to OWNxExplorer and does call the inherited method.